tfx/components/example_gen/utils.py

```diff
@@ -101,36 +121,16 @@ def dict_to_example(instance: Dict[str, Any]) -> example_pb2.Example:
     if isinstance(pyval, bytes):
       pyval = pyval.decode(_DEFAULT_ENCODING)
 
-    if pyval is None:
```

Comment:
Actually, this case was not handled. So I revived `if pyval is None` conditional. Thanks!